tests/test_get.py

```diff
@@ -45,21 +47,23 @@ def test_get_calls_correct_url(mocker):
     except ValueError:
```

Comment:
This one too?
```suggestion
    except requests.exceptions.HTTPError:
```

Reply:
I'm not sure why this code is wrapped with `try/except`. It's not meant to be testing exception handling (and if one happened it was ignoring it). I removed the `try/except` and the test still passes, so I think that's a small improvement.